core/lib/src/main/java/org/opennms/core/daemon/DaemonReloadEnum.java

```diff
@@ -30,6 +30,8 @@
 
 public enum DaemonReloadEnum {
 
+    ALARMD("Alarmd"),
```

Comment:
Looks like the names for these Daemons are in lower case `alarmd` and `collectd`.

Reply:
You're right